contrib/generic_proxy/filters/network/source/proxy.cc

```diff
@@ -382,6 +390,11 @@ Envoy::Network::FilterStatus Filter::onData(Envoy::Buffer::Instance& data, bool)
   return Envoy::Network::FilterStatus::StopIteration;
 }
 
+void Filter::onDecodingSuccess(RequestPtr request, ExtendedOptions options, bool end_stream) {
+  ENVOY_LOG(debug, "response end stream[{}]", end_stream);
+  onDecodingSuccess(std::move(request), options);
+}
```

Comment:
So, we still create a new ActiveStream for every request frame here, right?

Reply:
Yes, the new ActiveStream need to be created for every request frame